### PR TITLE
Vets api parthur change spool emails

### PR DESCRIPTION
**Pull request by PJart**
## Description of change
Added Accenture employees to spool errors, staging 10203 spool submissions, staging spool submissions, staging year to date report, flipper admin user emails. Adjusted tests that used settings.

### CODE REVIEW

**Review by AdamFreemer — COMMENTED** [review]
Just need extra lines at the end of indicated files.

File: Gemfile.lock [hunk 1]
```diff
@@ -1149,4 +1149,4 @@ RUBY VERSION
    ruby 2.6.6p146
 
 BUNDLED WITH
-   1.17.3
+   1.17.3
```
Comment (af-manish) [inline Gemfile.lock:1152]: We shouldn't have Gemfile.lock and there needs to be a newline at end of file.

File: config/settings.yml [hunk 2]
```diff
@@ -1092,4 +1133,4 @@ bid:
 
 # Settings for Test User Dashboard modules
 test_user_dashboard:
-  env: dev
+  env: dev
```
Comment (af-manish) [inline config/settings.yml:1136]: We need a newline at end of file.

File: Gemfile.lock (hunk 1 above)
Comment (af-manish) [reply Gemfile.lock:1152]: Actually I think Gemfile.lock got added because there's no newline. If you add new line, it won't consider it changed.

**Review by af-manish — APPROVED** [review]

**Review by rileyanderson — APPROVED** [review]

**Pull request merged**